perf_amd64.go

```diff
@@ -0,0 +1,7 @@
+package perf
+
+// doEnableRunDisable enables the counters, executes f, and disables them. It is
+// implemented in assembly to minimize non-deterministic overhead. It is assumed
+// that perfFD is known to be a valid file descriptor at the time of the call,
+// no error checking occurs.
+func doEnableRunDisable(perfFD uintptr, f func())
```

Comment:
Was this `go fmt`-ed? GH says there is no newline at the end of this file.

Reply:
Good spot. My editors auto-fmt functionality is borked :(